web3sTests/Multicall/MulticallTests.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MulticallTests.swift
+//  web3swiftTests
+//
+//  Created by David Rodrigues on 28/10/2020.
+//  Copyright © 2020 Argent Labs Limited. All rights reserved.
+//
+
+import XCTest
+@testable import web3swift
+
+class MulticallTests: XCTestCase {
+    var client: EthereumClient!
+    var multicall: Multicall!
+    let testContractAddress = EthereumAddress(TestConfig.erc20Contract)
+
+    override func setUp() {
+        super.setUp()
+        self.client = EthereumClient(url: URL(string: TestConfig.clientUrl)!)
+        self.multicall = Multicall(client: client!)
+    }
+
+    func testNameAndSymbol() throws {
+        var aggregator = Multicall.Aggregator()
+
+        try aggregator.append(ERC20Functions.name(contract: testContractAddress))
+        try aggregator.append(ERC20Functions.symbol(contract: testContractAddress))
+
+        let expect = expectation(description: "Get token name and symbol")
+        multicall.aggregate(calls: aggregator.calls) { result in
+            do {
+                switch result {
+                case .failure(let error):
+                    XCTFail("Multicall failed with error: \(error)")
+                case .success(let response):
+                    let name = try response.outputs[0].value
```

Comment:
I'm evaluating how to improve the way we pass the output to consumers to allow a better handling as right now they need to correlate the order of each call with the expected output and do the respective parsing of the response.

Reply:
Improved in https://github.com/argentlabs/web3.swift/pull/116/commits/47cede40dd314180c09b490961d9355f64e1427e.